typescript/infra/config/environments/mainnet3/warp/configGetters/getEthereumBscLumiaLUMIAWarpConfig.ts

```diff
@@ -38,7 +38,7 @@ export const getEthereumBscLUMIAWarpConfig = async (
     mailbox: '0x3a867fCfFeC2B790970eeBDC9023E75B0a172aa7',
     proxyAdmin: {
       owner: owner,
-      address: '0xeA87ae93Fa0019a82A727bfd3eBd1cFCa8f64f1D',
```

Comment:
@yorhodes - Wanted to confirm that you agree that this was a misconfiguration. `0xeA87ae93Fa0019a82A727bfd3eBd1cFCa8f64f1D` is the AW Hyperlane proxyAdmin on Lumia see https://github.com/hyperlane-xyz/hyperlane-registry/pull/348/files#diff-db9320e038aa13af3ce7b3297aab2dcc286e31c2bb4d3f656a2e91d4d5194e0cL14